kimchi/src/linearization.rs

```diff
@@ -68,6 +68,10 @@ pub fn constraints_expr<F: PrimeField + SquareRootField>(
         expr += xor::Xor16::combined_constraints(&powers_of_alpha);
     }
 
+    if rot {
+        expr += rot::Rot64::combined_constraints(&powers_of_alpha);
```

Comment:
I would stick with "common interface pattern" like in range-check and foreign_field_mul, so that one day we can more easily update the code to use a Gadget/gate interface and all this code could be generic, i.e. do the same thing for each switched on gate.  There's no reason to hard code and duplicate all this code really.